ehttp/src/types.rs

```diff
@@ -28,6 +30,20 @@ impl Request {
         }
     }
 
+    #[cfg(feature = "multipart")]
+    pub fn multipart(url: impl ToString, builder: MultipartBuilder) -> std::io::Result<Self> {
+        let (content_type,data) = builder.finish()?;
+        Ok(Self {
+            method: "POST".to_string(),
```

Comment:
Wouldn't it be petter to pass the `method` by argument to also support multipart PUT for instance?

Reply:
I think it would make the most sense to have post as default and a special function like this to change it.
```
 pub fn method(mut self, method: Method) -> Self {
        self.method = method.to_string();
        self
}
```
https://github.com/JustFrederik/ehttp/tree/custom_method

Reply:
Should I add it? I have a json branch. Should I add it or not

Reply:
Sure, you can add it, but then it should be called `with_method` imho.